### ISSUE TITLE
P2: husband field — 5 false-positive predictions in fine-tuned Document AI extractor

### ISSUE THREAD

**Issue by danyelajunebrown** [danyelajunebrown/Reparations-is-a-real-number#46]
## Problem

The fine-tuned `Freedmans_Bank_Deposit_Reader` extractor (evaluated 2026-04-28) makes 5 predictions for the `husband` field across the 24-doc test set, **all of them wrong**.

```
"husband": {
    "f1Score": 0,
    "precision": 0,
    "recall": 0,
    "predictedInstancesCount": 5,
    "predictedDocumentsCount": 4,
    "groundtruthInstancesCount": 3,
    "groundtruthDocumentsCount": 1
}
```

The base `pretrained-foundation-model-v1.5` already had F1 0 on this field but only over-predicted by 1. The fine-tune *increased* the over-prediction count.

## Why this matters less than #45 (last_mistress)

Pipeline doesn't currently consume `husband` for any enslaver inference path. But:
- It DOES populate `unconfirmed_persons.relationships` with spouse data
- A wrong husband attribution corrupts spouse relationship records
- If we later cross-reference Freedmen's Bank spouses to canonical_persons (e.g., issue #41 depositor promotion), wrong husbands propagate

## Mitigation

1. **Disable the field in the consumer** until Document AI improves it — don't write `husband` to `unconfirmed_persons.relationships` from the new extractor's output
2. **Or apply confidence floor** — but no `optimalConfidence` was identifiable from metrics (model has no signal threshold that helps here)
3. **Or label more training data** — only 1 ground-truth document with 3 instances. Model can't learn from one example.

## Acceptance

- [ ] Decision on consumer behavior (disable / threshold / label more data)
- [ ] If consumer still writes husband: confidence threshold applied
- [ ] Track in next training round whether more labeled examples lift this

Filed from Apr 27, 2026 Document AI evaluation. Related: #45 (last_mistress precision).
